btree/btree.c

```diff
@@ -2139,9 +2129,9 @@ static void ff_make(struct slot *slot, struct m0_be_tx *tx)
 	M0_PRE(ff_rec_is_valid(slot));
 	M0_PRE(ff_isfit(slot));
 	memmove(start + rsize, start, rsize * (h->ff_used - slot->s_idx));
-	mem_update(node, tx, start + rsize, sizeof *(start + rsize));
+	//m0_be_tx_capture();
 	h->ff_used++;
-	mem_update(node, tx, h, sizeof *h);
+	//m0_be_tx_capture();
```

Comment:
Mention here that this capture should note the node-header contents which have been modified.

Reply:
Updated in the latest commit.